components/train.yaml

```diff
@@ -106,6 +108,22 @@ implementation:
           ${bootleg_args} \
           $@
 
+      if test "$calibrate" == "true" ; then
+        genienlp predict \
+            --path "$modeldir" \
+            --data "dataset" \
+            --eval_dir "eval_dir" \
+            --task ${task_name} \
+            --silent \
+            --overwrite \
+            --evaluate valid \
+            --save_confidence_features \
+            --confidence_feature_path "$modeldir/confidence_feature_file.pkl" \
```

Comment:
Do we want to save the feature file and upload it to S3? Otherwise, we should write somewhere else.
How big is it? I imagine I don't want to download it for every model I donwload locally, because it's not useful unless debugging.

Reply:
It is in the order of 100 MB. It is useful for calibration experiments.